### DISCUSSION TITLE
Door/window binary sensor not shown correctly in HA

### DISCUSSION THREAD

**Discussion by MattFromGer** [zwave-js/zwave-js#6191] [Request Support / Investigate Issue] [answered]
I added a [door/window sensor](https://github.com/zwave-js/node-zwave-js/blob/master/packages/config/config/devices/0x0175/mt02648.json) to my network, but HA is getting wrong sensors transmitted from zwave-js.
HA with unknown sensor values:
<img width="532" alt="HA" src="https://github.com/zwave-js/node-zwave-js/assets/4983643/b50dd56b-cb98-4aac-a905-d54d31b0b61a">

In zwave-js UI I've got the correct binary sensor, but also the "notification" sensors with undefined values.
<img width="835" alt="zwavejs UI" src="https://github.com/zwave-js/node-zwave-js/assets/4983643/f3f4a88e-0364-43bf-8cb0-35bd3b67665c">

The logs show that the binary sensor is the one which is changing.
```
2023-08-18T14:10:56.779Z SERIAL » [ACK]                                                                   (0x06)
2023-08-18T14:10:56.781Z CNTRLR   [Node 012] [Binary Sensor] Door/Window: metadata updated          [Endpoint 0]
2023-08-18T14:10:56.783Z CNTRLR   [Node 012] [~] [Binary Sensor] Door/Window: false => true         [Endpoint 0]
2023-08-18T14:10:56.784Z DRIVER « [Node 012] [REQ] [ApplicationCommand]
                                  └─[BinarySensorCCReport]
                                      type:  Door/Window
                                      value: true
```

So who is at fault here?

**Comment by kpine** [comment, marked as answer]
In your list of disabled sensors will be a binary sensor that is mapped to Binary Sensor CC Report. Enable that and use it instead of the others. Binary Sensor is a "legacy" CC and was replaced by Notification CC, so the latter is preferred in HA.

Parameter 7 controls whether the device uses Binary Sensor CC or Notification CC. What is yours set to? This is described in the user manual. It sounds like notification CC reports are disabled. If you want to use the default-enabled entities, you would need to configure it to use notification CC. Probably easier to use the other entity.

**Reply by MattFromGer** [reply]
Thank you so much for your time and detailed explanation! Both solutions worked. Since binary is legacy, I chose to set the parameter 7 from binary type to notification. Further I wondered why there is a duplicate "simple" sensor, but you answered that already [here](https://github.com/home-assistant/core/issues/88060#issuecomment-1501217131). All clear, thanks again! :)

**Reply by kpine** [reply]
It's fine to use the "legacy" sensor FYI, it's just deprecated in the Z-Wave specs, but will continue to function. Some devices support both, probably for older software.
